tests/subsys/canbus/isotp/implementation/src/main.c

```diff
@@ -203,7 +203,7 @@ static void receive_test_data(struct isotp_recv_ctx *recv_ctx,
 		      "Expected timeout but got %d", ret);
 }
 
-static void test_send_receive_net_sf(void)
+ZTEST(isotp_tests, test_send_receive_net_sf)
```

Comment:
```suggestion
ZTEST(isotp_implementation test_send_receive_net_sf)
```

Same here. You have renamed the test suite, but failed to rename any of the test cases. This will result in none of the test being executed. You should see this quite clearly when validating your changes.